src/pages/authentication/EnterPinPage.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { useCallback, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { Typography, Button, Dialog, DialogTitle, DialogContent, DialogActions, Stack, Snackbar, Alert } from '@mui/material';
+import Logo from '../../components/Logo/Logo';
+import MinimalWrapper from '../../layout/MinimalLayout/MinimalWrapper';
+import PinInput from '../../pages/authentication/PinInput';
+import CenteredLayout from './CenteredLayout';
+
+const EnterPinPage: React.FC = () => {
+  const [open, setOpen] = useState<boolean>(false);
+  const [pin, setPin] = useState<string[]>(() => Array(4).fill(''));
+  const [openSnackbar, setOpenSnackbar] = useState<boolean>(false);
+  const navigate = useNavigate();
+
+  const handleNextClick = () => {
+    if (pin.every(p => p !== "")) {
+      //TODO: Implement the logic to check the PIN
+      navigate('/enter-pin');
```

Comment:
can we navigate to home page here?

Reply:
Yes, it's typo. It should be navigate('/');